flutter/lib/src/sentry_redaction_options.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
+import 'package:meta/meta.dart';
+
+import '../sentry_flutter.dart';
+import 'screenshot/masking_config.dart';
+import 'screenshot/widget_filter.dart';
+
+/// Configuration of the experimental redaction feature.
+class SentryRedactingOptions {
```

Comment:
WDYAT about naming this `SentryPrivacyOptions` and the name in options.experimental.privacy?

Reply:
yeah, I like this idea.